### PR TITLE
[Keyboard] Pachi restructure and keyboard addition

### PR DESCRIPTION
**Pull request by Xelus22**
## Description
Restructure to move the keymaps inside the respective boards.

Adds Pachi RGB keyboard.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Move keymaps into their own sub folders
* Add Pachi RGB

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by drashna** [comment]
This should probably target develop.

**Review by drashna — APPROVED** [review]

**Comment by Xelus22** [comment]
Does this require develop? I have Pachi RGB delivering within the month :(

File: keyboards/xelus/pachi/rgb/rules.mk [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+EEPROM_DRIVER = i2c
+
+RGB_MATRIX_ENABLE = yes
+RGB_MATRIX_DRIVER = custom
+
+COMMON_VPATH += $(DRIVER_PATH)/issi
+SRC += drivers/issi/is31fl3741.c
+
+LTO_ENABLE = yes
+OPT = 2
```
Comment (Erovia) [inline keyboards/xelus/pachi/rgb/rules.mk:32]:
```suggestion
OPT = 2
```

Does `O2` provide smaller/faster code than the default `Os`?

**Review by Erovia — CHANGES_REQUESTED** [review]
Can you remove the empty layers from the `default` keymaps and add reboot instructions to the revision `readme.md`'s as per the [PR-checklist](https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs)?

File: keyboards/xelus/pachi/rgb/rules.mk (hunk 1 above)
Comment (Xelus22) [reply keyboards/xelus/pachi/rgb/rules.mk:32]: Yes, O2 is faster code.

**Review by Erovia — APPROVED** [review]

**Pull request merged**